puffin/src/lib.rs

```diff
@@ -566,6 +560,13 @@ pub fn now_ns() -> NanoSecond {
     START_TIME.0 + START_TIME.1.elapsed().as_nanos() as NanoSecond
 }
 
+#[inline]
+#[cfg(all(target_arch = "wasm32", not(feature = "web")))]
+pub fn now_ns() -> NanoSecond {
+    // This should be unused.
+    0
```

Comment:
I personally prefer loud error (panic) to "why the hell doesn't my profiler work!?!?!"

Reply:
Makes sense too, yeah. I've done that now.